meet-data/raw/raw-probe

```diff
@@ -60,27 +49,5 @@ def getmeetlist(html):
 
     return urls
 
-
-def getenteredurls():
-    urls = []
-    for dirname, subdirs, files in os.walk(FEDDIR):
-        if 'URL' in files:
-            with open(dirname + os.sep + 'URL', 'r') as fd:
-                for k in fd.readlines():
-                    urls.append(k.strip())
-    return urls
-
-
-def main():
-    html = gethtml(URL)
-    meetlist = getmeetlist(html)
-
-    known = getenteredurls()
-
-    for m in meetlist:
-        if not m in known:
-            print(color('[RAW] ') + m)
-    print(color('[RAW] ') + "Continue working through archive.")
```

Comment:
The printing logic should probably remain in this file.

Although there's some additional logic to the printing that would probably be nice to have in the helper library -- specifically, some scripts accept a `--quick` argument that changes their printing behavior a little. It would be nice to standardize the usage of that argument.